src/core/ComboBox/ComboBox.tsx

```diff
@@ -136,6 +136,10 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
   }, [isOpen]);
 
   const [filteredOptions, setFilteredOptions] = React.useState(options);
+  React.useEffect(() => {
```

Comment:
Comment mentions added `useLayoutEffect`, but here its not that one.

Reply:
Yeah I changed it as it does not have an *effect* 😆